### ISSUE TITLE
 JupyterLab access to scalable computing ressources

### ISSUE THREAD

**Issue by huard** [Ouranosinc/pavics-sdi#63]
Register workers on boreas and/or CC resources to parallelize job (not necessarily WPS process).

**Comment by tomLandry** [comment]
A brief exploration of Pangeo capabilities on that topic would be beneficial.

**Comment by huard** [comment]
Agreed, I've already started playing with xarray+dask, but for the whole "cloud storage" thing we'll need to provision cloud space for testing.
